### ISSUE TITLE
Show timestamp in request history

### ISSUE THREAD

**Issue by johanmickos** [a85/POSTMan-Chrome-Extension#1315]
It would be convenient and useful to have a clearly visible timestamp label for each request sent in the History tab.

**Comment by aames** [comment]
+1 for this - even if it's a premium only feature it'd be useful.

**Comment by yarivat** [comment]
+1

**Comment by Hiblet** [comment]
This would be very useful.  Use-case: I am working on an API with a Developer, I send a Request as part of a test series.  Later, a problem is found that relates to the Request, and the Developer needs to check logs to see what happened in her app around the time of receiving my request.  I would look at my history tab and see the send timestamp (ideally in UTC).  At the moment, I have to guess the send time.

So, +1.

**Comment by vkaegis** [comment]
Hey guys, with the last release, you can nor see the user who triggered a history request as well as the timestamp for it. Closing this issue, feel free to open another if you have more feedback around this.

**Comment by udayraj-rzp** [comment]
@vkaegis can you reopen this? We need this feature

**Comment by joneiros** [comment]
This feature currently exists, but is not "clearly visible" as is requested by @johanmickos.
How to see currently:
- When you hover over a given row, your profile avatar is displayed:
<img width="174" alt="history tab item with avatar showing" src="https://user-images.githubusercontent.com/14276371/231866894-bef91da0-ce70-443b-84f0-1395514de25a.png">
- When you hover over that little avatar, the date and time are displayed:
<img width="282" alt="history tab item profile hover showing name (masked) and datetime" src="https://user-images.githubusercontent.com/14276371/231867973-88fe6edf-0b2c-47a5-8877-a1176b7cc490.png">
(Edit - This part is outdated) ~~ Note that the hover functionality seems finicky, and I found I needed to hover on the avatar, then hover a few pixels to the left, and then back on to get the datetime display to show up consistently. ~~

---
Hopefully that helps for people (like me) who didn't immediately find the current implementation.
Environment Details:
- Postman Destop App for Mac V10.12.10
<img width="237" alt="image" src="https://user-images.githubusercontent.com/14276371/231869009-32fc8eeb-fc1c-43f2-a8a7-e931a854d44b.png">
